.github/workflows/gin.yml

```diff
@@ -31,21 +45,21 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ github.ref }}
-
-      - name: Install Dependencies
-        run: make tools
-
-      - name: Run Check
-        run: |
-          make vet
-          make fmt-check
-          make misspell-check
```

Comment:
This is replaced by golangci-lint job. I didn't modified the Makefile so it can still be done manually